st2common/st2common/runners/utils.py

```diff
@@ -78,3 +82,38 @@ def get_action_class_instance(action_cls, config=None, action_service=None):
         action_instance.action_service = action_service
 
     return action_instance
+
+
+def invoke_post_run(liveaction_db, action_db=None):
```

Comment:
Let's please add some standalone "unit" tests for this functions for all the scenarios and edge cases:

* action_db is None and action exists in db
* action_db is None and action doesn't exist in db